external/libtorch_gpu.BUILD

```diff
@@ -4,14 +4,15 @@ licenses(["notice"])
 
 cc_library(
     name = "libtorch_gpu",
+    srcs = glob(["lib/*.so"]),
+    hdrs = glob(["*"]),
     includes = [
-        ".",
-        "torch/csrc/api/include",
+        "include",
+        "include/torch/csrc/api/include",
+        "include/torch/csrc/api/include/torch",
     ],
     linkstatic = False,
-    linkopts = [
```

Comment:
I prefer this form. As libtorch was pre-installed, and we don't need them  until the linking stage.

Reply:
You are right, and I find that it seems like python3.7 is no longer needed in this way...